imagespace_smqtk/server/smqtk_search.py

```diff
@@ -23,28 +23,23 @@
 from girder.plugins.imagespace import solr_documents_from_field
 
 from .settings import SmqtkSetting
+from .utils import base64FromUrl
 
 import json
 import requests
-import os
 
 DEFAULT_PAGE_SIZE = 100
 NEAR_DUPLICATES_THRESHOLD = -1500  # Maximum distance to be considered a near duplicate
 
 class SmqtkSimilaritySearch(Resource):
     def __init__(self):
         setting = SmqtkSetting()
-        self.search_url = setting.get('IMAGE_SPACE_SMQTK_SIMILARITY_SEARCH')
+        self.search_url = setting.get('IMAGE_SPACE_SMQTK_SIMILARITY_SEARCH') + '/nn'
```

Comment:
It looks like you are removing this value (`IMAGE_SPACE_SMQTK_SIMILARITY_SEARCH`) and replacing it with `IMAGE_SPACE_SMQTK_NNSS_URL` in other places. Is it supposed to still be here?